src/App.jsx

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
```

Comment:
- [ ] I get an error from here kindly fix it 
![image](https://github.com/haftamudesta/React-Capstone-Project/assets/62705947/2a6db921-d0fc-4e91-abdc-fdbe9ad573e8)

Reply:
✖️  There's a new error for the Navbar CSS module import that still needs resolving...